src/style.css

```
@@ -1 +1,42 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  position: relative;
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

can be a bit simplify
```suggestion
  background: url(images/star.svg) center no-repeat;
```